src/locales/pl.json

```diff
@@ -0,0 +1,101 @@
+    "tos": "Warunki użytkowania",
+    "tos_content": "<p><b>Use of \"VeganCheck.me\"</b></p><p>The use of the site \"VeganCheck.me\" is free of charge. The distribution of the contents or our services against payment is strictly prohibited. Automated queries are also prohibited and may only be performed using the <a href=\"https://jokenetwork.de/vegancheck-api\">API</a>.<br> We reserve the right to change, reduce, expand or disable our services at any time.</p><p><b>Processing of entered texts and numbers</b></p><p>On our website, barcodes or ingredients of a product can be entered in text fields. This is necessary to provide our service in its entirety. The data entry is completely voluntary.<br>Only barcodes or ingredients may be entered in the named text fields; the entry or transmission of personal data of any kind is strictly prohibited.</p><p>Personal data is any information that relates to an identified or identifiable living individual. Different pieces of information, which collected together can lead to the identification of a particular person, also constitute personal data.<sup><a href=\"https://ec.europa.eu/info/law/law-topic/data-protection/reform/what-personal-data_en\">?</a></sup></p><p><b>Disclaimer</b></p><p>We assume no liability for the accuracy of the product information and ingredient lists offered and for the availability of our service.</p>"
+  },
+  "Privacy": {
+    "germanonly": "Ta strona jest dostępna tylko w języku niemieckim."
+  },
+  "404": {
+    "error404": "Błąd 404.",
+    "pagedoesnotexist": "Ta strona nie istnieje.",
+    "message": "Sprawdź naszą {statuspage} albo nasz profil w sieci {mastodon} aby dowiedzieć się o aktualnych problemach.",
+    "statuspage": "stronę statusu"
+  },
+  "Fallback": {
+    "yourareoffline": "Brak połączenia z siecią.",
+    "needinternet": "Aby użyć VeganCheck.me połącz się z internetem.",
+    "tryagain": "Spróbuj ponownie"
+  },
+  "InstallPrompt": {
+    "subheading": "Czy To Wegańskie? - Vegan Check",
+    "install": "Zainstaluj VeganCheck.me",
+    "get": "Pobierz",
+    "howtoinstall": "Wybierz {share} a następnie \"Dodaj do ekranu głównego\"."
+  },
+  "ShortcutPrompt": {
+    "Shortcuts": "Skróty",
+    "openinapp": "Otwórz w aplikacji Skróty",
+    "open": "Otwórz"
+  },
+  "Ingredients": {
+    "ingredientcheck": "Weryfikator Składników Wegańskich",
+    "ingredientcheck_desc": "Sprawdź, czy składniki produktu są wegańskie! Wprowadź składniki oddzielając je przecinkiem",
+    "entercommaseperated": "Wprowadź składniki oddzielone przecinkiem",
+    "submit": "Wyślij",
+    "vegan": "Wegańskie",
+    "source": "Źródło danych",
+    "cannotbeempty": "Lista składników nie może być pusta, zawierać znaków specjalnych i musi być oddzielona przecinkiem!",
+    "licenses": "Licencje",
+    "licenses_desc": "VeganCheck.me uses different databases and APIs to gather information about a product. The information is licensed under the following licenses:",
```

Comment:
You're right, there's no need to translate this, as it's only legal stuff. Good one!